tests/test_annotated_types.py

```diff
@@ -281,3 +281,34 @@ class Model(BaseModel):
 
     with pytest.raises(ValidationError):
         Model.parse_obj({'t': {'v': -1}})
+
+
+def test_typeddict_annotated_nonoptional():
+    class DataTD(TypedDict):
+        a: Optional[int]
+        b: Annotated[Optional[int], Field(...)]
+        c: Annotated[Optional[int], Field(..., description='Test')]
+        d: Annotated[Optional[int], Field()]
+
+    class Model(BaseModel):
+        data_td: DataTD
```

Comment:
Can we add some tests for actually using the model, not just generating a schema?

Reply:
Can try, although all I had interest originally was the schema itself.